tasks/bundle_analysis_processor.py

```diff
@@ -68,33 +67,28 @@ def run_impl(
                 retry_num=self.request.retries,
             ):
                 return self.process_impl_within_lock(
-                    db_session=db_session,
-                    repoid=repoid,
-                    commitid=commitid,
-                    commit_yaml=commit_yaml,
-                    params=deepcopy(params),
```

Comment:
I see we got rid of deepcopy here but I don't see any writing/changes to anything inside of it. I'm curious why we had this in the first place 🤔